applications/nrf5340_audio/src/bluetooth/le_audio.h

```diff
@@ -127,13 +127,24 @@ struct le_audio_evt {
 typedef void (*le_audio_receive_cb)(const uint8_t *const data, size_t size, bool bad_frame,
 				    uint32_t sdu_ref);
 
+enum le_audio_button_id {
```

Comment:
It would be nice to not create a new enum for the buttons. I think we can use "button_pin_names" in button_assignments.h?